src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,23 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import cn from 'classnames';
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../types/Todo';
+import { getUserById } from '../../App';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const getUser = getUserById(todo.userId);
```

Comment:
- `user` is a more suitable name
- And actually better just to store information about a user in todo, and just pass it through props. As this component is presentation component and shouldn't have any logic